### DISCUSSION TITLE
Is the "Stream Deck +" Supported?

### DISCUSSION THREAD

**Discussion by sallymanda** [streamdeck-linux-gui/streamdeck-linux-gui#92] [Q&A]
Hey everyone :-) Just wondering if the "Stream Deck +" is supported, as it isn't listed as a supported device in the description.

**Comment by dhtseany** [comment]
Not yet, we're still waiting for upstream to merge a commit that will
provide initial support for the device. Once that happens we'll be able to
start working on the front end support for the Stream Deck +.

https://github.com/abcminiuser/python-elgato-streamdeck/issues/105#issuecomment-1749009887
